frontend/packages/core-auth/src/utils/getSessionStorage.ts

```diff
@@ -0,0 +1,6 @@
+export const getSessionStorage = (key: string): any | undefined => {
```

Comment:
This name should indicate that it's getting JSON. Maybe, `getJSONFromSessionStorage`?